lib/workflowmgr/slurmbatchsystem.rb

```diff
@@ -300,8 +300,9 @@ def submit(task)
 
           queued_jobs,errors,exit_status=WorkflowMgr.run4("squeue -u #{username} -M all -t all -O jobid:40,comment:32", 45)
 
-          # Raise SchedulerDown if the command failed
-          raise WorkflowMgr::SchedulerDown,errors unless exit_status==0
+          # Don't raise SchedulerDown if the command failed, otherwise
+          # jobs that have moved to sacct will be missed.
+          #raise WorkflowMgr::SchedulerDown,errors unless exit_status==0
```

Comment:
Also please change the rescue clause on 310 to remove the catching of `WorkflowMgr::SchedulerDown` since it is no longer being raised.